3-binary-data/lib/images.js

```diff
@@ -67,16 +67,14 @@ function sendUploadToGCS (req, res, next) {
 // [END process]
 
 // Multer handles parsing multipart/form-data requests.
-// This instance is configured to store images in memory and re-name to avoid
-// conflicting with existing objects. This makes it straightforward to upload
-// to Cloud Storage.
+// This instance is configured to store images in memory.
+// This makes it straightforward to upload to Cloud Storage.
 // [START multer]
-var multer = require('multer')({
-  inMemory: true,
-  fileSize: 5 * 1024 * 1024, // no larger than 5mb
-  rename: function (fieldname, filename) {
```

Comment:
Multer has it built-in now.

> By default, Multer will rename the files so as to avoid naming conflicts. The renaming function can be customized according to your needs.
- https://github.com/expressjs/multer